templates/meilisearch/files/search/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+install_meilisearch() {
+    # Check Platform.sh variable, to vary installation based on local v. Platform.sh
+    echo "* INSTALLING MEILISEARCH"
+    # Replicates Meilisearch download (https://github.com/meilisearch/MeiliSearch/blob/master/download-latest.sh) with locked version.
+    release_file="meilisearch-linux-amd64"
+    curl -OL "https://github.com/meilisearch/MeiliSearch/releases/download/$MEILISEARCH_VERSION/$release_file"
+    mv "$release_file" "meilisearch"
+    chmod 744 "meilisearch"
+}
+
+setup_venv(){
+    echo "* SETTING UP POETRY VENV"
+    # Install poetry
+    curl https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py >> get-poetry.py
+    python get-poetry.py --version $POETRY_VERSION
+    # Source the Poetry command and cleanup.
+    . $PLATFORM_APP_DIR/.poetry/env && rm get-poetry.py
+    # Add Poetry to .bash_profile, so available during SSH.
+    echo ". $PLATFORM_APP_DIR/.poetry/env" >> ~/.bash_profile
```

Comment:
Shouldn't this rather go in `.environment`?

Reply:
Should be.
That's where we put env vars for nvm when installing an alternative node version to make this alternative version available during SSH. So this should work just the same?